tests/test_presentation_p4_family_629.py

```
@@ -0,0 +1,575 @@
+    blob = "\n".join([
+        str(scene.get("scene_text") or ""),
+        str(scene.get("origin_context") or ""),
+        str(scene.get("criterion_text") or ""),
+        str(scene.get("gap_text") or ""),
+    ])
+    assert "与喀尔喀而治" not in blob
+    assert diegetic in blob
+
+
+def test_urge_lever_due_review_import_order_both_succeed():
+    """负向：先 import urge_lever 与先 import due_review 两种乱序均成功（无环烟）。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)